pep/app/pods/components/modal-dialogs/user/about/template.hbs

```diff
@@ -2,14 +2,34 @@
     <Modal.header @title={{t "about.title"}} />
     <Modal.body>
         <div class="mb-2">
-            <h5 class="mb-0">{{t "about.client"}}</h5>
-            <p class="text-muted small">{{@options.clientBuildVersion}}</p>
+            <h5 class="mb-0">
+                {{t "about.status.title"}}
+                {{#if this.isStatusOk}}
+                    <FaIcon @icon="check-circle" @prefix="fal" class="text-success"  />
+                {{else}}
+                    <FaIcon @icon="exclamation-circle" @prefix="fal" class="text-danger" />
```

Comment:
I dont think you need `@prefix="fal"` on any of these icons, as thats the default prefix.